test/e2e/data/infrastructure-aws/kustomize_sources/topology/cluster-template.yaml

```diff
@@ -0,0 +1,29 @@
+---
+apiVersion: cluster.x-k8s.io/v1beta1
```

Comment:
Not sure if the folder layout is okay (especially placing the ClusterClass here). It works, but please let me know if something else would be better.